src/main/java/com/wolox/training/models/Book.java

```diff
@@ -69,6 +69,21 @@ public Book() {
     users = new ArrayList<>();
   }
 
+  public Book(String genre, String author, String image, String title, String subtitle,
+      String publisher, String year, Integer pages, String isbn,
+      List<User> users) {
+    this.genre = genre;
+    this.author = author;
+    this.image = image;
+    this.title = title;
+    this.subtitle = subtitle;
+    this.publisher = publisher;
+    this.year = year;
+    this.pages = pages;
+    this.isbn = isbn;
+    this.users = users;
```

Comment:
You should use your setters here instead of this assignation. This way you assure that the preconditions are accomplished :) 
I mean, instead of `this.author = author;` use `this.setAuthor(author);`